src/ui/components/atoms/SecondaryButton/SecondaryButton.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import { useState, useCallback } from 'react';
+import type { SecondaryButtonProps } from './types/SecondaryButtonProps';
+import './SecondaryButton.scss';
+
+function SecondaryButton({
+  size = 'medium',
+  disabled = false,
+  onClick,
+  active = false,
+  focused = false,
+  hovered = false,
+}: SecondaryButtonProps) {
+  const [isHovered, setIsHovered] = useState(hovered);
```

Comment:
Why you need this states ?

Reply:
Using these states allows you to create a more interactive and accessible button component that responds to different user interactions from both mouse and keyboard events. 
These states are useful for applying different CSS styles using the generated class (classNames).